### PR TITLE
added lines to temp file

### PR DESCRIPTION
**Pull request by rahulkesagani**

### CODE REVIEW

**Comment by avinashgandham** [comment]
Thanks
Add few more lines possible

File: temp [hunk 1]
```diff
@@ -1,2 +1,5 @@
 creating temp file
 which will be very useful in the later stages
+yes you ar right
+this is a very important file
+please consider these changes
```
Comment (avinashgandham) [inline temp:5]: Thanks for making changes

Comment (avinashgandham) [inline temp:5]: Thanks for making these changes 
please add one more line if possible

Comment (rahulkesagani) [reply temp:5]: sure

**Comment by avinashgandham** [comment]
thanks

**Comment by rahulkesagani** [comment]
u r welcome

**Pull request merged**